src/renderers/server-render.jsx

```diff
@@ -187,24 +191,13 @@ const makeRenderer = (
 	})
 	.first(state => state.preRenderChecklist.every(isReady => isReady));  // take the first ready state
 
-	request.log(['app', 'info'], `Finding route for path: '${appLocation}'`);
-	return Rx.Observable.bindNodeCallback(match)({ location: appLocation, routes })
-		.do(([redirectLocation, renderProps]) => {
-			if (!redirectLocation && !renderProps) {
-				throw Boom.notFound();
-			}
-		})
-		.do(() => {
-			request.log(['app', 'info'], 'Found app route, dispatching RENDER');
-		})
-		.do(([redirectLocation, renderProps]) =>
-			store.dispatch({
-				type: '@@server/RENDER',
-				payload: renderProps.location
-			})
-		)
-		.flatMap(args => storeIsReady$.map(() => args))  // `sample` appears not to work - this is equivalent
-		.map(getRouterRenderer(store, baseUrl, clientFilename, assetPublicPath));
+	request.log(['app', 'info'], `Finding route for path: '${url}'`);
+	store.dispatch({
+		type: '@@server/RENDER',
+		payload: request.url,
+	});
+	return storeIsReady$
+		.map(() => getRouterRenderer(routes, store, url, baseUrl, clientFilename, assetPublicPath));
```

Comment:
Could you explain how this works now? When we were pairing you were saying that the consumer apps would need to pass in their `AppContainer` components to makeRenderer, but I'm not seeing that now.